src/plugins/data_source_management/public/components/drop_down_header/_drop_down_header.scss

```diff
@@ -0,0 +1,3 @@
+#invisibleFocusable {
```

Comment:
Can we follow the dev guide for scss to change the naming and also scope the id down to under the component? https://github.com/opensearch-project/OpenSearch-Dashboards/blob/main/DEVELOPER_GUIDE.md#sass-files

Reply:
thx, followed the guide the remove rename file to  remove the `_` prefix. and updated the class name to be more specific